src/test/datascience/jupyter/interpreter/jupyterInterpreterSubCommandExecutionService.unit.test.ts

```diff
@@ -84,6 +87,9 @@ suite('DataScience - Jupyter InterpreterSubCommandExecutionService', () => {
         when(interperterService.getActiveInterpreter()).thenResolve(activePythonInterpreter);
         when(interperterService.getActiveInterpreter(undefined)).thenResolve(activePythonInterpreter);
     });
+    teardown(() => {
+        sinon.restore();
```

Comment:
Apparently `sinon.restore` is [deprecated and internal](https://github.com/sinonjs/sinon/issues/1390). We're supposed to create a sinon.Sandbox instance and create the stub with `sandbox.stub`, then restore with `sandbox.restore`. I guess if this works it's fine though?